scripts/transfer-doc-files.js

```diff
@@ -22,14 +23,17 @@ function toTwoDigits(value) {
  * @returns {string}
  */
 function frontMatterTemplate(componentName) {
+    const title = componentName === '_index' ?
+        'Components for Game User Interface' :
+        componentName.charAt(0).toUpperCase() + componentName.slice(1);

```

Comment:
Can we also replace dashes from the name with spaces so that we get Automatic grid and not Automatic-grid?

Reply:
Yes. Added this replacement